lib/rubocop/cop/mixin/preceding_following_alignment.rb

```diff
@@ -55,7 +55,7 @@ def aligned_with_line?(line_nos, range, indent = nil)
         false
       end
 
-      def comment_lines
```

Comment:
I ran into another method called `comment_lines` that is moved to the new module. I thought this may be a place to swap in the new method, but this `comment_lines` takes alignment into account. So I gave it more descriptive name.